docs/REST API/rest-api.md

```diff
@@ -7404,21 +7404,60 @@ This allows you to easily configure scheduled queries that will impact a whole t
 ## Scripts
 
 - [Run script](#run-script)
-- [Get script result](#get-script-result)
-- [Run live script](#run-live-script)
 - [Add script](#add-script)
-- [Delete script](#delete-script)
+- [Get script result](#get-script-result)
+- [Upload script](#upload-script)
+- [Download script](#download-script)
 - [List scripts](#list-scripts)
-- [Get or download script](#get-or-download-script)
-- [Get script details by host](#get-script-details-by-host)
+- [Delete script](#delete-script)
 
 ### Run script
 
-Run a script on a host.
+This action will execute a script on the host if no other scripts are currently running. If script actions are not complete within 5 minutes of execution the script process will terminate.
+
+To add a script to a host's list of upcoming activities use the [Add script](#add-script) API endpoint.
 
-The script will be added to the host's list of upcoming activities.
+`POST /api/v1/fleet/scripts/run/sync`
 
-The new script will run after other activities finish. Failure of one activity won't cancel other activities.
+#### Parameters
+
+| Name            | Type    | In   | Description                                      |
+| ----            | ------- | ---- | --------------------------------------------     |
+| host_id         | integer | body | **Required**. The host id to run the script on.  |
+| script_id       | integer | body | The ID of the existing saved script to run. Only one of either `script_id`, `script_name` or `script_contents` can be included in the request; omit this parameter if using `script_contents` or `script_name`.  |
+| script_contents | string  | body | The contents of the script to run. Only one of either `script_contents`, `script_id` or `script_name` can be included in the request; omit this parameter if using `script_id` or `script_name`. |
+| script_name       | string | body | The name of the existing saved script to run. Only one of either `script_name`, `script_id` or `script_contents` can be included in the request; omit this parameter if using `script_contents` or `script_id`.  |
+| team_id       | integer | body | ID of the team the saved script referenced by `script_name` belongs to. Default: `0` (hosts assigned to "No team") |
+
+
+> Note that if both `script_id` and `script_contents` are included in the request, this endpoint will respond with an error.
+
+#### Example
+
+`POST /api/v1/fleet/scripts/run/sync`
+
+##### Default response
+
+`Status: 200`
+
+```json
+{
+  "host_id": 1227,
+  "execution_id": "e797d6c6-3aae-11ee-be56-0242ac120002",
+  "script_contents": "echo 'hello'",
+  "output": "hello",
+  "message": "",
+  "runtime": 1,
+  "host_timeout": false,
+  "exit_code": 0
+}
+```
+
+### Add script
```

Comment:
Not sure the "add script" naming makes it clear that this runs a script (elsewhere in the API docs "Add" means creating a new record). @noahtalerman thoughts?

Reply:
This can be anything, I was trying to make it simple. I am afraid of the word queue, again, because in this context it doesn't really have a clear meaning, i.e., I came to the API to run a script, why is there an option for queuing it?

Reply:
It looks like we moved the "Add script" header to the wrong endpoint. 

Today, "Add script" is for `POST /api/v1/fleet/scripts` (docs [here](https://fleetdm.com/docs/rest-api/rest-api#add-script)). This endpoint creates a new record.

Reply:
I think it's hard to say that it's wrong. I am willing to do this as many times as needed to get it right, but, the problem in my opinion is that the docs as they exist are not clear. That's what I am trying to address.

Reply:
Brock! We appreciate you digging into the docs and suggesting changes. Thank you.

> there are conventions you are following. I am not sure if they are being consistently applied.

There are conventions. Currently, we want to use "Add" for every endpoint that adds a new record. However, maybe it makes sense to use "Upload" instead if the endpoint is asking for a file.

Can you please bring this PR to product office hours today? We can discuss and align on the conventions.